cloudinstall/charms/compute.py

```diff
@@ -48,6 +49,8 @@ class CharmNovaCompute(CharmBase):
     allow_multi_units = True
     allowed_assignment_types = [AssignmentType.BareMetal,
                                 AssignmentType.KVM]
+    is_core = True
+    depends = ['nova-cloud-controller', 'mysql', 'glance', 'rabbitmq']
```

Comment:
this and the change to controller.py raise a question - for the 'core' set of charms, we don't need to put them in each others' depends lists, since they will all be deployed together anyway.
It might be worth leaving them in as documentation, but I'd vote to leave them out since they might get out of sync because they're not really necessary, and not cause problems until further down the line.